gui/app/templates/interactive/effect-options/cooldown.html

```diff
@@ -0,0 +1,24 @@
+<div class="effect-setting-container">
+    <div class="effect-specific-title"><h4>Which buttons should I put on cooldown?</h4></div>
+    <div class="cooldown-group-buttons">
+        <ul class="cooldown-group-buttons-list">
+            <li ng-repeat="button in boardButtons">
+                <input ng-click="buttonArray(effect.buttons,button)" ng-checked="buttonCheckboxer(effect.buttons,button)" type="checkbox" aria-label="..." >
+                <span>{{button}}</span>
+            </li>
+        </ul>
+    </div>
+    <div class="cooldown-group-button-reset">
+        <button class="btn btn-warning">Uncheck all</button>
```

Comment:
I took a look at this and I'm not sure what the best way to handle it is yet. All the examples I found put ng-model="all" on the uncheck button and then ng-checked="all" on the inputs. But, if I do this then I'll need to replace the function currently there which checks the correct boxes on load.

Recommendations?